tools/cloud-build/daily-tests/ansible_playbooks/tasks/wait-for-startup-script.yml

```diff
@@ -15,17 +15,17 @@
 - name: Assert variables are defined
   ansible.builtin.assert:
     that:
-    - vm_name is defined
     - timeout_seconds is defined
 
 - name: Wait for startup script to complete
   become: true
-  ansible.builtin.wait_for:
-    path: /var/log/messages
-    search_regex: '.*{{ vm_name }}.*startup-script exit status ([0-9]+)'
-    timeout: "{{ timeout_seconds }}"
-  register: startup_status
+  ansible.builtin.shell: journalctl -u google-startup-scripts.service -b | grep -oP "startup-script exit status \K\d+"
+  register: result
+  until: result.rc == 0
+  retries: "{{ (timeout_seconds/5)|int }}"
```

Comment:
3 hr of this will generate over 2000 lines of logs.

Reply:
I don't love it but after looking for 30 minutes I don't see an alternative. I am going to approve and merge so we can see the test pass tonight. We can iterate if we don't like the logging.

Reply:
There's a way to remove the log on a per-step basis, I was just worried that the entire step wouldn't be logged.  We could just increase the time step to a minute or more and that will cut the log lines significantly.  The extra minute of wait time shouldn't be too much of a hassle on other faster tests.